react-app/src/store/reviews.js

```diff
@@ -0,0 +1,105 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const initialState = { entities: { reviews: {} } }
+
+export const createReview = createAsyncThunk(
+    'reviews/createReview',
+    async (itemId, formDetails, thunkAPI) => {
```

Comment:
The parameters `itemId` and `formDetails` need to be in an object.
Change this to `({itemId, formDetails}, thunkAPI)`